prefect_aws/credentials.py

```diff
@@ -21,10 +19,20 @@ class AwsCredentials:
             This is only needed when you are using temporary credentials.
         profile_name: The profile to use when creating your session.
         region_name: The AWS Region where you want to create new connections.
+
+    Example:
+        Load stored AWS credentials:
+        ```python
+        from prefect_aws import AwsCredentials
+
+        aws_credentials_block = AwsCredentials.load("MY_BLOCK_NAME")
     """  # noqa E501
 
+    _block_type_name = "AWS Credentials"
+
+
     aws_access_key_id: Optional[str] = None
-    aws_secret_access_key: Optional[str] = None
+    aws_secret_access_key: Optional[SecretStr] = None
     aws_session_token: Optional[str] = None
```

Comment:
Should this be a SecretStr too? Not too familiar with AWS

Reply:
I do not believe so, as I think it is useless without the aws_secret_access_key